crates/driver/src/domain/competition/solution/settlement.rs

```diff
@@ -282,12 +282,6 @@ impl Settlement {
                     mempools::RevertProtection::Disabled => gas_cost,
                 };
                 let objective_value = quality - gas_cost;
-                if objective_value.is_zero() {
-                    return Err(score::Error::RiskAdjusted(
-                        score::risk::Error::ObjectiveValueNonPositive,
-                    ));
-                }
-
```

Comment:
I was looking at this commit and I don't know why this was removed or where it's handled now. Is it redundant now?

Reply:
This was moved to `score()` function, and with https://github.com/cowprotocol/services/pull/2022/commits/39858e03f8c9b0bcdec8ead269ae5f094584db78 is moved to `Sub` implementation (Quality - GasCost)